### PR TITLE
Feature/77 UIs look nice for all screen sizes and screen rotations

### LINKED ISSUE #77: Modify UI's to be able to use the app on phones and tablets in portrait or landscape mode

### LINKED ISSUE #115: Pictures are automatically cropped so that they aren't distored

### PR DESCRIPTION
**Pull request by NicolasVial**
Add : 

- Tablet compatibility for all UIs
- Compatibility with screen rotations (for phone and tablet)
- Refactored CategoryEditing UI

Comment : 

- I didn't touch MainActivity since we are always modifying it. Tell me if you want it to be cleaner and "tablet + rotation proof".

### CODE REVIEW

**Review by Nitwix — COMMENTED** [review]
Looks all good from a code point of view, but I need more time to test the UI. Will approve when I had the time to test this

File: app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/CategoriesEditingActivityTest.kt [hunk 1]
```diff
@@ -129,12 +130,13 @@ class CategoriesEditingActivityTest {
...
         )
     }
-    
+
     @Test
     fun addNewCategoryToDatasetWorks() {
         val nbCategories = dataset.categories.size
         onView(withId(R.id.button_add)).perform(click())
         onView(withText("")).perform(typeText("new beautiful category"))
+        closeSoftKeyboard()
```
Comment (lourkeur) [inline app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/CategoriesEditingActivityTest.kt:139]: :clap:

**Review by bbjubjub2494 — APPROVED** [review]
appears to work well on a landscape-mode phone, nothing wrong with the code. Good job as far as I am concerned. please fix the merge conflict.

**Comment by Nitwix** [comment]
There is a crash in the learning presenter when there is no next picture in the dataset. I'll push a quick fix shortly

**Review by Nitwix — APPROVED** [review]
Good modifications! I couldn't test on a tablet but it looks much cleaner overall!

**Pull request merged**